app/models/proposal.rb

```diff
@@ -69,6 +70,40 @@ class Proposal < ActiveRecord::Base
     where(created_at: start_time...end_time)
   }
 
+  # elasticsearch indexing setup
+  DEFAULT_INDEXED = {include: {client_data: {}, comments: {}, steps: {}}}
+
+  settings index: {
+    number_of_shards: 1, # increase this if we ever get more than N records
+    number_of_replicas: 1
+  } do
+    # with dynamic mapping==true, we only need to explicitly define overrides.
```

Comment:
this comment seems important but I might need a bit more context to full understand it...perhaps we should put it in the docs?